promql/testdata/functions.test

```diff
@@ -734,3 +843,69 @@ eval instant at 5m absent_over_time({job="ingress"}[4m])
 
 eval instant at 10m absent_over_time({job="ingress"}[4m])
 	{job="ingress"} 1
+
+clear
+
+# Testing exp() sqrt() log2() log10() ln()
+load 5m
+	vector_matching_a{l="x"} 0+1x100
```

Comment:
You only need one timestamp to test these.

Reply:
Eliminiated.

Reply:
You removed the series, not the timestamps which is the x100

Reply:
My mistake, easily reverted.

Reply:
So I re-added all the series, and I reduced the amount of timestamps. It's not super clear from the test themselves why they should or shouldn't be there. I could reduce it to a single one many of the answers will need to be updated as a result.

Reply:
Also I did simplify it down to a single datapoint. Just recently